lib/fileutils.rb

```diff
@@ -631,16 +631,52 @@ def link_entry(src, dest, dereference_root = false, remove_destination = false)
   end
   module_function :link_entry
 
+  # Copies files from +src+ to +dest+;
+  # +src+ may not be the path to a directory
```

Comment:
This second part, describing what src and dest may not be seems weird.  I would just drop the second part, and change the semicolon to a period.  Near the end of method documentation, state that if +src+ is a directory, an exception is raised.  You can completely remove the part about dest not being a path to a file, since that is allowed.

Reply:
Done.